README.md

```diff
@@ -45,6 +45,25 @@ In addition to command line parameters, all configuration options support enviro
 
 **Note**: Command line parameters take precedence over environment variables.
 
+## Installation
+
+### Nix Package Manager
+
+With Nix package manager, you can easily install wg-ddns:
+
+- Install from GitHub repository
+```bash
```

Comment:
We may need an `enter` here to create a new empty line, so that we can make sure our documentation will show as same on some special markdown preview tool. Like:

```
- Install from GitHub
<empty line here>
<code block and something else>
```